src/pages/authentication/building-todo-app.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Building a Todo app
-description: Learn how to integrate authentication and data storage.
+title: Review Todos app
```

Comment:
I'm a bit puzzled by this - why would a developer come here to review an app? I think the goal should be to build/deploy an app. Could we redesign this tutorial to include active steps / changes devs would have to make throughout the tutorial?

I'd suggest we fork a branch of the todo app that has comments in several places where changes should be made to make the app work. The branch would be like a boilerplate, ready to be completed during the tutorial

@hstove - thoughts?

Reply:
I'd prefer to get these fixes in place before we open up the question of whether this "review" type of material vs. "build" is better (and reworking the entire structure if we pursue the latter, which I agree could have greater merit). Currently this is the only way in which our docs exhibit Stacks.js-based authentication for Connect clearly, so it pains me to leave them essentially broken and misleading while we work out higher level questions.

Also keep in mind that after getting this merged, it's top priority for me to put in place a build-like tutorial sequence for auth, storage and transaction signing. That fresh effort may eventually replace this todos list app entirely, though I do think code samples with guidance have value.

Reply:
I'm not sure I get it. Why would someone look for a review of an app in the journey of learning about a library?

I think there's a low hanging fruit to introduce some changes to make this a more meaningful tutorial, without reworking everything. The content in here can still be used mostly.

Just pushing this out at this stage won't be impactful I believe. It'll allow us to say there's content but not that there's useful content. I'd suggest we make small corrections to make this a real tutorial.

Reply:
Someone may very well want to look at how a simple app is built with a certain technology or library IMO. This is personally how I like to learn new tech, fwiw. But I also agree it's not necessarily the only or best way.

I think what you're proposing is that we basically make the user copy and paste snippets into a more partial todos code base, which I'm concerned would just make busy work for them (since copy and pasting doesn't really make them think). We could make it more engaging and complicated, and I do think that's worth exploring, but that's going to take time. So in the meantime, I don't see why it hurts getting this merged, feeling confident our docs aren't broken, then starting that effort (immediately)?

Reply:
We had a convo in discord and I'll put a tl;dr in here: if you believe the content (as-is) will have the impact you're looking for on the product side, I'm fine merging it. 

Outside of this particular instance, I would urge us not to get comfortable with a "review" type of tutorial in other places.

Reply:
I would agree that this tutorial is actually a review of how to build an app vs a step by step implantation from scratch.

Reply:
Thanks 🙏  I agree, let's shoot higher than "code review" for content structure going forward.